libafl_qemu/src/emu/usermode.rs

```diff
@@ -22,7 +22,7 @@ impl<C, CM, ED, ET, I, S, SM> Emulator<C, CM, ED, ET, I, S, SM> {
     }
 
     #[must_use]
-    pub fn access_ok(&self, kind: VerifyAccess, addr: GuestAddr, size: usize) -> bool {
+    pub fn access_ok(&self, kind: VerifyAccess, addr: GuestAddr, size: usize) -> Option<bool> {
```

Comment:
At _some_ point you could start adding comments :D

Reply:
It's somewhat impossible to know when this would return None just from looking at the fn name